scripts/release-post/index.mjs

```diff
@@ -208,6 +212,16 @@ const renderPost = results => {
   return { content: template(templateParameters), ...results };
 };
 
+const formatPost = results => {
+  return new Promise((resolve, reject) => {
+    format(results.content, { ...prettierConfig, parser: 'markdown' })
+      .then(formattedContent =>
```

Comment:
Can we call this variable just content? I think that's fine haha

Reply:
Updated in 8b5c81c